js/plugins/googleai/src/gemini.ts

```diff
@@ -209,6 +209,23 @@ export const gemini20ProExp0205 = modelRef({
   configSchema: GeminiConfigSchema,
 });
 
+export const gemini20FlashThinkingExp0121 = modelRef({
```

Comment:
this also needs to be exported from the index.ts

Reply:
Done